api4jenkins/job.py

```diff
@@ -119,6 +119,15 @@ def get_scan_log(self, stream=False):
                              stream=stream) as resp:
             yield from resp.iter_lines()
 
+    def build(self, branch, **params):
```

Comment:
there already have function `scan' to trigger build, see https://github.com/joelee2012/api4jenkins/pull/60/files#diff-1ee6cccfbabcd51d6691b7bb3074afe486a443318dec89408d6e02e1965b621fR114

Reply:
Thanks for the asnwer. Indeed it's functionality I was missing.